src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -60,10 +59,11 @@ class CallActor(call: Call, locallyQualifiedInputs: Map[String, WdlValue], backe
       context.parent ! CallFailed(call, message)
     }
 
+    // FIXME: Perhaps this could be renamed to "launchCall" or something like that?
```

Comment:
@mcovarr - I'm happy to change this myself but what do you think of the suggested name? `handleSuccessfulInstantiation` doesn't really describe what it does :(